### PR TITLE
Mobile debug console

### PR DESCRIPTION
**Pull request by Velenir**
To alleviate the pain of debugging on mobile in cases when can't debug as a remote device in Chrome devtools (not every in-app browser is recognized by Chrome).

A mobile-only console component

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr418--dexreact.review.gnosisdev.com

**Comment by W3stside** [comment]
Happy to keep this for now but we should move it into some external repo for future use, will test more tomorrow morning. But nice work!

File: src/App.tsx [hunk 1]
```diff
@@ -92,6 +93,9 @@ const App: React.FC = () => (
         </Switch>
       </Layout>
     </Router>
+    {process.env.NODE_ENV === 'development' &&
+      Web3Connect.isMobile() &&
```
Comment (alfetopito) [inline src/App.tsx]: I tried on mobile but wasn't able to see the console.
What do I need to turn it on?

Comment (Velenir) [reply src/App.tsx]: did you try it in development or opened the pr418-- link ? the link is deployed in production

Comment (alfetopito) [reply src/App.tsx]: The link from this PR (https://pr418--dexreact.review.gnosisdev.com)

Comment (W3stside) [reply src/App.tsx]: @alfetopito He's saying those PR links are always `NODE_ENV='production'` so it won't work from there

Comment (alfetopito) [reply src/App.tsx]: Ahhhhh got it. So I need to spin this up in local and then open the browser? Giving it a try...

Comment (alfetopito) [reply src/App.tsx]: Saw it now, very cool

**Pull request merged**